packages/subscription-manager/src/components/TabsSwitcher/styles.scss

```diff
@@ -0,0 +1,63 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@wordpress/base-styles/breakpoints";
```

Comment:
The breakpoints import is not necessary here. It was me who added it here in our draft PR 🙈 I will remove it. 🙂